### PR TITLE
Feature - restore index

### PR DESCRIPTION
**Pull request by lyazii22**
- [ ] Restore index implementation

### CODE REVIEW

File: ui/src/app/components/object-detail/object-detail.component.html [hunk 1]
```diff
@@ -67,11 +67,29 @@ <h3 class="title">
         color="primary"
         class="icon drop"
         (click)="dropIndex()"
-        *ngIf="currentObject!.isSpannerNode && currentObject!.type === 'indexName'"
+        *ngIf="
```
Comment (shreyakhajanchi) [inline ui/src/app/components/object-detail/object-detail.component.html:70]: The count of indexes should get modified once the index is deleted : https://drive.google.com/file/d/1yEcLrr0Ybhpa1HdgTPxkMGP373u98qJG/view?usp=share_link&resourcekey=0-1gymAPFb0NCsB4Q8ZSGEWQ

**Review by shreyakhajanchi — COMMENTED** [review]
LGTM mostly, left few minor comments

File: ui/src/styles.scss [hunk 2]
```diff
@@ -280,23 +284,27 @@ body {
...
 
-
-.sidenav-title{
-  margin-top :10px;
+.sidenav-title {
+  margin-top: 10px;
   margin-bottom: 0px;
 }
+<<<<<<< HEAD
+=======
```
Comment (shreyakhajanchi) [inline ui/src/styles.scss:307]: Please resolve the merge conflicts before checking the code in or marking the PR ready for review.

Comment (Ronak-Searce) [reply ui/src/styles.scss]: Will take care of this.

**Review by shreyakhajanchi — APPROVED** [review]
LGTM

**Pull request merged**